src/components/Poster/Poster.js

```diff
@@ -3,10 +3,12 @@ import './Poster.css'
 
 const Poster = ({id, title, rating, imgSrc, selectMovie}) => {
   return (
-    <div className="poster" onClick={() => selectMovie(id)}>
+    <div className="poster" tabIndex='0' onClick={() => selectMovie(id)}>
       <img src={imgSrc}/>
-      <h2>{title}</h2>
-      <p>{rating.toFixed(1)}/10 Fresh Tomatillos</p>
+      <div>
+        <p>Freshness: {rating.toFixed(2)*10}%</p>
+        <h2>{title}</h2>
+      </div>
```

Comment:
This is a great Idea, I will need to go back into the movie page and do the same thing so we are consistent throughout the whole app.